code/modules/client/verbs/ghost.dm

```diff
@@ -23,8 +23,8 @@ GLOBAL_LIST_INIT(ghost_verbs, list(
 	switch(alert("Descend to the Underworld?",,"Yes","No"))
 		if("Yes")
 			if(istype(mob, /mob/living/carbon/spirit))
-				//HONEYPOT CODE, REMOVE LATER
-				message_admins("RETARDED MOTHERFUCKER [key] IS TRYING TO CRASH THE SERVER BY SPAWNING 3 GORILLION SPIRITS!")
+				//HONEYPOT CODE, REMOVE LATER (Maybe I will, maybe I won't.)
+				message_admins("[key] soul has rejoined the underworld.")
```

Comment:
whut

Reply:
Figure out if this is still an issue and if not just remove it- otherwise adjust the message to be proper

Reply:
Nah, I know what this is for, is when someone disconnects while in the underworld, and their entity gets de-spawned, but once they connect its entity gets spawned in the overworld, which triggers the message, then their entity gets dragged back into the underworld, no biggie really.